README.md

```diff
@@ -24,7 +24,7 @@ In order to keep these files versionned, we use symlinks from this directory tow
 When you edit one of these files, run this command afterwards so that your changes are taken into account:
 
 ```sh
-service nginx reload
+sudo systemctl restart nginx.service
```

Comment:
Keep reloading command as it's safer for example, if a syntax error exist in the modified config file? 
(the server remains running)

Reply:
Yep, restarting is rarely necessary and is likely to create downtime. Strongly advice against using this regularly.